EddystoneBeacon.cpp

```diff
@@ -76,6 +78,18 @@ void EddystoneBeacon::begin(char power, const char* uri) {
 
   this->_bleCharacteristic.broadcast();
 }
+/***Following is the code for broadcasting TLM ****/
+/***Implementation in the examples folder***/
+void EddystoneBeacon::begin(const unsigned char* temp, unsigned char len){
```

Comment:
A nicer API would be to specify the battery voltage and temperature as numbers, instead of raw TLM data.